packages/contextual-help/src/ContextualHelp.ts

```diff
@@ -0,0 +1,205 @@
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import {
+    CSSResultArray,
+    html,
+    render,
+    SpectrumElement,
+    TemplateResult,
+} from '@spectrum-web-components/base';
+import '@spectrum-web-components/action-button/sp-action-button.js';
+import '@spectrum-web-components/overlay/sp-overlay.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-help-outline.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-info-outline.js';
+import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
+import {
+    property,
+    state,
+} from '@spectrum-web-components/base/src/decorators.js';
+import {
+    removeSlottableRequest,
+    SlottableRequestEvent,
+} from '@spectrum-web-components/overlay/src/slottable-request-event.js';
+import type { Placement } from '@spectrum-web-components/overlay/src/overlay-types.js';
+import {
+    IS_MOBILE,
+    MatchMediaController,
+} from '@spectrum-web-components/reactive-controllers/src/MatchMedia.js';
+
+import styles from './contextual-help.css.js';
+
+/**
+ * Spectrum Contextual help provides additional information about
+ * the state of either an adjacent component or an entire view.
+ * @element sp-contextual-help
+ *
+ * @slot Text content to display in the popover
+ * @slot link - link to additional informations
+ */
+export class ContextualHelp extends SpectrumElement {
+    protected isMobile = new MatchMediaController(this, IS_MOBILE);
+
+    public static override get styles(): CSSResultArray {
+        return [styles];
+    }
+
+    /**
+     * Optional title to be displayed inside the popover.
+     * @param {String} headline
+     */
+    @property()
+    public headline?: string;
+
+    /**
+     * Provides an accessible name for the action button trigger.
+     * @param {String} label
+     */
+    @property()
+    public label?: string;
+
+    /**
+     * The `variant` property applies specific styling on the action button trigger.
+     * @param {String} variant
+     */
+    @property()
+    public variant: 'info' | 'help' = 'info';
+
+    /**
+     * @type {"top" | "top-start" | "top-end" | "right" | "right-start" | "right-end" | "bottom" | "bottom-start" | "bottom-end" | "left" | "left-start" | "left-end"}
+     * @attr
+     */
+    @property({ reflect: true })
+    public placement?: Placement = 'bottom-start';
+
+    /**
+     * The `offset` property accepts either a single number, to
+     * define the offset of the Popover along the main axis from
+     * the action button, or 2-tuple, to define the offset along the
+     * main axis and the cross axis.
+     */
+    @property({ type: Number })
+    public offset: number | [number, number] = 0;
+
+    @state()
+    overlayOpen = false;
```

Comment:
You probably want to normalize on `open` and make it an `@property()` so that consuming apps can declare that the help is open when needed.